Riku/src/GameLogic/Assets/Asset.cpp

```diff
@@ -8,6 +8,7 @@
 #include <filesystem>
 #include <fstream>
 #include <hash-library/sha256.h>
+#include "../../../Riku/TestMove.h"
```

Comment:
Jak do tego doszło, że potrzeba aż tylu znaków ..?

Reply:
w sumie dobre pytenie

Reply:
TestMove przez przypadek stworzył mi się w złym folderze. Wtedy to olałem bo stwierdziłem że i tak będę go zaraz usuwał, ale potem się okazało że na razie zostaje i o tym zapomniałem xD. Przeniosę go tam gdzie powinien być